rpc/src/rpc.rs

```diff
@@ -379,16 +380,15 @@ impl JsonRpcRequestProcessor {
             .tpu(connection_cache.protocol())
             .unwrap();
         let (sender, receiver) = unbounded();
-        SendTransactionService::new::<NullTpuInfo>(
+
+        let client = ConnectionCacheClient::<NullTpuInfo>::new(
+            connection_cache.clone(),
             tpu_address,
-            &bank_forks,
             None,
-            receiver,
-            connection_cache,
-            1000,
```

Comment:
Why 1000? Maybe some comment explaining  this number?

Reply:
This is pre-existing code used only in tests, I think it is magic number which is selected from common sense considerations. Will check this out in follow ups.